packages/date-picker/src/vaadin-date-picker-overlay-content.js

```diff
@@ -289,11 +209,26 @@ class DatePickerOverlayContent extends ControllerMixin(ThemableMixin(DirMixin(Po
       _todayButton: {
         type: Object,
       },
+
+      calendars: {
+        type: Array,
+        value: () => [],
+      },
+
+      years: {
+        type: Array,
+        value: () => [],
+      },
     };
   }
 
   static get observers() {
-    return ['__updateCancelButton(_cancelButton, i18n)', '__updateTodayButton(_todayButton, i18n, minDate, maxDate)'];
+    return [
+      '__updateCalendars(calendars, i18n, minDate, maxDate, selectedDate, focusedDate, showWeekNumbers, _ignoreTaps, _theme)',
```

Comment:
New observers are needed to propagate properties previously handled by templatizer `forwardHostProp`.